Pages/Index.cshtml

```diff
@@ -114,7 +114,7 @@
     </div>
     <br/>
     <hr />
-
+    </main>
```

Comment:
Closed the tag in Index.cshtml because Closing Tag was not available.

Reply:
Huh.  I'll confess this was a new one to me.  https://www.w3schools.com/tags/tag_main.asp